clang/lib/DPCT/TypeLocRewriters.cpp

```diff
@@ -91,37 +91,29 @@ class CheckTypeNameAndInVarDecl {
 };
 
 inline std::function<bool(const TypeLoc TL)>
-checkTemplateArgSpelling(size_t index, std::string str) {
+checkTemplateArgSpelling(size_t Index, std::string Str) {
+  auto getQualtifiedNameStr = [=](const NamedDecl *NL) -> std::string {
+    if (const auto *NSD = dyn_cast<NamespaceDecl>(NL->getDeclContext())) {
+      std::string TypeQualifiedString =
+          getNameSpace(NSD) + "::" + NL->getNameAsString();
+      return TypeQualifiedString;
+    }
+    return "";
```

Comment:
If there is no qualifier, the type name will always be empty, is it expected?

Reply:
It is mistake. Fixed